### DISCUSSION TITLE
Issue in the Model Viewer with typescript

### DISCUSSION THREAD

**Discussion by samuelss1102** [community/community#20880] [Projects and Issues] [answered]
How can I implement the Google Model Viewer in Typescript?

**Comment by IceDev528** [comment, marked as answer]
I already have implement the google model viewer with this code.
Please try this and let me know if you have any question.
import '@google/model-viewer';

declare global {
namespace JSX {
interface IntrinsicElements {
'model-viewer': ModelViewerJSX &
React.DetailedHTMLProps<React.HTMLAttributes, HTMLElement>;
}
}
}
